rapture/transport/workers.py

```diff
@@ -109,17 +109,21 @@ def scp_func(settings, filename, results):
     address = settings['address']
     username = settings['username']
     port = settings.get('port', 22)
+    destination = settings['destination']
```

Comment:
So destination is a required field? If not, can you change it to a `settings.get` with a sane default path?